pkg/endpoint/bpf.go

```diff
@@ -765,6 +766,11 @@ func (e *Endpoint) regenerateBPF(owner Owner, currentDir, nextDir string, regenC
 		return 0, compilationExecuted, fmt.Errorf("unable to regenerate policy because PolicyMap synchronization failed: %s", err)
 	}
 
+	err = eppolicymap.WriteEndpoint(epInfoCache.keys, e.PolicyMap.Fd)
+	if err != nil {
+		e.logStatusLocked(BPF, Warning, fmt.Sprintf("Unable to sync EpToPolicy Map continue with Sockmap support: %s", err))
```

Comment:
with --> without?

Reply:
Its correct as is but may be confusing. Sockmap will continue to run but with a missing policy sockets residing in the endpoint will not be able to use sockmap.